README.md

```diff
@@ -29,5 +29,21 @@ curl http://localhost:3000/weather/Curitiba
 ```
 #### Response
 ```sh
-{"temperature":"14 °C","wind":"6 km/h"}
+{  
+   temperature:"24 °C",
```

Comment:
Thanks for contribution @nilemarbarcelos :grinning: 
Just don't forget to update the json without the format (tags between quotation marks).